src/config/configurations/sueDefaultConfigurations.ts

```diff
@@ -0,0 +1,74 @@
+const defaultApiConfig = {
+  apiKey: '',
+  areaService: { apiKey: '', id: '', serverUrl: '' },
+  jurisdictionId: '',
+  serverUrl: '',
+  testApi: { apiKey: '', jurisdictionId: '', serverUrl: '' },
+  whichApi: ''
+};
+
+const geoMapDefaultConfig = {
+  areas: [],
+  center: {},
+  clisterTreshold: 2,
```

Comment:
what is clister? or a typo for cluster?

Reply:
It should be `clusterThreshold`. It's a typo.

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/pull/1131/commits/4b4abafe9ad39a3435adf6e113da7b9ec15e7920